CMakeLists.txt

```diff
@@ -0,0 +1,362 @@
+# Copyright 2017 The LEVELDB Authors. All rights reserved.
+# Use of this source code is governed by a BSD-style license that can be
+# found in the LICENSE file. See the AUTHORS file for names of contributors.
+
+cmake_minimum_required(VERSION 3.1)
+
+list(APPEND CMAKE_MODULE_PATH "${CMAKE_SOURCE_DIR}/cmake")
```

Comment:
`CMAKE_SOURCE_DIR` is root directory of the sources. If this project will be used as a subproject then `${CMAKE_SOURCE_DIR}/cmake` will not mean directory relative to this file. See:
* https://cgold.readthedocs.io/en/latest/tutorials/cmake-sources/includes.html#cmake-current-list-dir-vs-cmake-current-source-dir

Reply:
OK